CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This is the log of notable changes to EAS CLI and related packages.
 
 ### 🎉 New features
 
+- Generate metadata file for project archive ([#2149](https://github.com/expo/eas-cli/pull/2149) by [@khamilowicz](https://github.com/khamilowicz))
```

Comment:
It would be good to rebase this PR and ensure that this changelog entry is not under some old CLI version published a while ago, but under the `main` section, because it's easy to forget about it when rebasing